### DISCUSSION TITLE
DatePicker access to min-date display error " Cannot read properties of null (reading 'getFullYear')"

### DISCUSSION THREAD

**Discussion by ghost** [formkit/formkit#1100] [General]
When i try to acces to years with lower of min-date "2023" in this case
    <FormKit
      type="datepicker"
      name="date"
      label="Conference date"
      placeholder="Select a date"
      help="When are you hosting your next conference?"
      picker-only
      min-date="2023-12-30T22:29:13.113Z"
    />
![date-picker](https://github.com/formkit/formkit/assets/99481334/93b1725a-b5c3-43ff-92b8-a8925f2816c7)

**Comment by justin-schroeder** [comment]
Can you provide a reproduction in the playground?

**Comment by ghost** [comment]
@justin-schroeder  done !
